templates/projects/annotations.html

```diff
@@ -0,0 +1,26 @@
+{% extends "projects/base.html" %}
+{% set active_page = "projects" %}
+{% set active_link  = "settings" %}
+{% set section = _('Configure Annotation Parameters') %}
+
+{% import "projects/_helpers.html" as helper %}
+{% from "_formhelpers.html" import render_field, render_checkbox_field, render_select2_field %}
```

Comment:
I don't see where `helper`, `render_checkbox_field` and `render_select2_field` are used. Let's remove them if they aren't used.